examples/game_gomoku/main.ipynb

```diff
@@ -156,38 +166,30 @@
     "        # Record the status of the game\n",
     "        self.game_end = False\n",
     "        \n",
-    "    def reply(self, x: dict = None) -> dict:\n",
-    "        if x is None:\n",
+    "    def reply(self, input_: dict = None) -> dict:\n",
```

Comment:
Why `x` is changed into `input_`? Since the base agent class `AgentBase` use `x` as input, it's better to keep them consistent